python/pyarrow/tests/parquet/test_dataset.py

```diff
@@ -1230,29 +915,22 @@ def test_ignore_custom_prefixes(tempdir, use_legacy_dataset):
                         partition_cols=['_part'])
 
     read = pq.read_table(
-        tempdir, use_legacy_dataset=use_legacy_dataset,
-        ignore_prefixes=['_private'])
+        tempdir, ignore_prefixes=['_private'])
 
     assert read.equals(table)
 
 
-@parametrize_legacy_dataset_fixed
-def test_empty_directory(tempdir, use_legacy_dataset):
-    # ARROW-5310 - reading empty directory
-    # fails with legacy implementation
+def test_empty_directory(tempdir):
```

Comment:
```suggestion
def test_empty_directory(tempdir):
    # ARROW-5310 - reading empty directory
```
Maybe keep the issue reference (that's still relevant for the dataset case, as that was the reason this test was introduced)